tutorials/config/daint.py

```diff
@@ -27,7 +27,23 @@
                     'launcher': 'srun',
                     'access': ['-C gpu', '-A csstaff'],
                     'environs': ['gnu', 'intel', 'nvidia', 'cray'],
-                    'max_jobs': 100
+                    'max_jobs': 100,
+                    'resources': [
+                        {
+                            'name': 'memory',
+                            'options': ['--mem={size}']
+                        }
+                    ],
+                    'container_platforms': [
+                        {
+                            'type': 'Sarus',
+                            'modules': ['sarus']
+                        },
+                        {
+                            'type': 'Singularity',
+                            'modules': ['singularity']
+                        }
+                    ]
```

Comment:
I have another config file that defines the containers: `daint-containers.py`. You should probably use that, because this will destroy the listings in the tutorial. I didn't like before that we were listing a whole big config that most of its parts were irrelevant to the context.

Reply:
Where is this configuration? I didn't find it